metadata/expireFilesMetadata/README.md

```diff
@@ -0,0 +1,62 @@
+Artifactory Expires Files Metadata User Plugin
+==============================================
+
+This plugin expires files when they are requested according to file pattern and expire delay stored in properties file.
+
+This plugin is inspired from [Expires Packages Metadata User Plugin](https://github.com/jfrog/artifactory-user-plugins/tree/master/metadata/expirePackagesMetadata).
+
+* Properties file format is Map where :
+    * the key is the repository key of the remote reposity
+    * the value is array where :
```

Comment:
While this is probably fine in this case, be aware that arrays like this in config files can turn into maintenance nightmares. We recently redesigned the `artifactCleanup` plugin to use maps instead of arrays, which solved a lot of headaches. Using a map here would allow for more optional fields to be introduced, and it would also allow for old fields to be properly deprecated and replaced (if necessary), without creating backwards compatibility issues or a lot of confusion.

Also, I feel like JSON is better suited to this type of data, but ultimately it doesn't matter much.

Reply:
ok, done!